tracing/src/lib.rs

```diff
@@ -174,6 +174,24 @@
 //! # fn main() {}
 //! ```
 //!
+//! For functions which don't have built-in tracing support and can't have
+//! the `#[instrument]` attribute applied (such as from an external crate,
+//! the [`Span` struct][`Span`] has a [`in_scope()` method][`in_scope`]
+//! which can be used to easily wrap synchonous code in a span.
+//!
+//! For example:
+//! ```rust
+//! # #[cfg(not(feature = "no-std"))]
```

Comment:
`tracing` doesn't have a feature called `no-std`; the standard library is disabled by _disabling_ the `std` feature. So, I would be
```rust
//! # #[cfg(not(feature = "std"))]
```
but, I don't actually think that will work. Feature flags are not passed to the compiler when building doctests.

Reply:
aaah sorry It was late in the day and I read the `Cargo.toml` wrong, lol, `no-std` is a category here not a feature...